src/commands/link/index.ts

```diff
@@ -109,6 +111,38 @@ export default class LinkIndex extends Command {
       throw new Error(chalk.red("No .git found in this directory. Please initialize a git repository with `git init`."));
     }
 
+    // Check if the current branch is 'main'
+    let currentBranch = "";
+    try {
+      currentBranch = execSync("git symbolic-ref --short HEAD", { encoding: "utf-8" }).trim();
+    } catch (error) {
+      this.log(chalk.red("Unable to determine the current branch. Are you inside a Git repository?"));
+      throw error;
+    }
+
+    if (currentBranch !== "main") {
+      this.log(chalk.red("You must be on the 'main' branch to link your repository."));
+      this.log("Please switch to the 'main' branch:");
```

Comment:
can we add chalk color to these, and simplify the multi-line logs, separate them by newlines etc

Reply:
I think only the error should be red. If it's all red, it's hard to read...

Reply:
And making it multi line comment would also make the code harder to read...
<img width="479" alt="image" src="https://github.com/user-attachments/assets/229d80bf-af20-407c-ad94-04f4c3f276c2">

Reply:
i'll just make them all red

Reply:
i still don't think they should all be red. CLI instructions, for example in `git`, are often plain color.
Making them all red actually makes the message less readable.

![image](https://github.com/user-attachments/assets/d0d6466e-782a-4850-b78b-4dbb09b611b3)